test/backend/test_onnx_backend.py

```diff
@@ -87,6 +87,10 @@ def skip_not_implemented_ops_test(test):
 # skip resize downsample with mode=linear
 backend_test.exclude(r'test_resize_downsample_linear[a-z,_]*')
 
+# range is using loop in the unit test but
+# loop is not supported in tf-onnx converter
```

Comment:
Do you mean tf-onnx or onnx-tf?

Reply:
same as for onehot.py. this change is not supposed to be part of this PR. fixed now.